_migration-assistant/deploying-migration-assistant/configuration-options.md

```diff
@@ -170,6 +170,47 @@ The `serviceSigningName` can be `es` for an Elasticsearch or OpenSearch domain,
 
 All of these authentication options apply to both source and target clusters.
 
+## Bring your own snapshot options
+
+An existing snapshot can alternatively be used to perform metadata as well as backfill migrations, instead of using the Migration Assistant to create a snapshot.
```

Comment:
At this point, do we want to specify non-incremental snapshots, or are we okay with either kind?

Reply:
After discussing, we should be able to handle both but want to add further testing for incremental snapshots in our repo